### DISCUSSION TITLE
Github website access

### DISCUSSION THREAD

**Discussion by imanadeko** [community/community#53357] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I got another laptop a few days ago.  I've been trying to access github.com so that I can log-in but it keeps saying site cannot be reached. Meanwhile, I can access github just fine on my old laptop and both laptops are connected to the same network. All other sites are working just fine on my new laptop except guthub.com

**Comment by cohhi** [comment]
what is ``guthub.com``?

**Reply by cohhi** [reply]
Please try resetting the network

**Comment by NJacksonDev** [comment]
It is likely an internet connectivity issue with your new laptop and not the github site. I would look into that first.
